src/content/en/2021/privacy.md

```diff
@@ -5,572 +5,541 @@ description: Privacy chapter of the 2021 Web Almanac covering adoption and impac
 
 <p class="note">Note that 40 breaches in the previous year is a lower bound, since many breaches are only discovered, or made public, several months after they have occurred.</p>
 
 
 {{ figure_markup(
   image="data_breaches_pwned_websites_per_date_and_breach_sensitivity.png",
   caption="Data breaches in the previous year and their sensitivity",
-  description="Chart that shows the number of domains involved in a breach over the course of the previous year and the sentitivity of the data breach.",
+  description="Chart that shows the number of domains involved in a breach over the course of the previous year and the sensitivity of the data breach. For Not Sensitive breaches it was 4 in August 2020, 2 in September, 6 in October, 1 in November, 6 in December, 5 in January 2021, 3 in February, 5 in March, 2 in April 1 in May, 1 in June, and finally 1 in July. For Sensitive breaches it's just 1 in February 2021 and 2 in March 2021.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=2029950563&format=interactive",
   sheets_gid="1435927653",
   sql_file="data_breaches_pwned_websites_per_date_and_breach_sensitivity.sql"
   )
 }}
 
+{# TODO - do we have anything to say about this chart? Perhaps that it's a year-round problem and that the recent tail off is probably more indiciative of under reporting as some breaches will not have been discovered and/or made public yet? If we don't have anything to say then should we just drop the chart? #}
 
 {{ figure_markup(
   image="data_breaches_pwned_accounts_per_class.png",
   caption="Number of impacted accounts in breaches per data class",
-  description="Bar chart showing the number of user accounts involved in data breaches, according to the data class that leaked in the breach.",
+  description="Bar chart showing the number of user accounts involved in data breaches, according to the data class that leaked in the breach. 641 million Email addresses	were included in breaches, 428 million Passwords, 369 million Names, 173 million Geographic locations, 149 million Phone numbers, 149 million Genders, 134 million Social media profiles, 127 million Education levels, 126 million Job titles, and finally 110 million Physical addresses.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=1612339126&format=interactive",
   sheets_gid="1158689200",
   sql_file="data_breaches_pwned_accounts_per_class.sql"
   )
 }}
 
+{# TODO - are email addresses _always_ leaked? Or only in HaveIBeenPwned's dataset since that is the primary key they use? #}
```

Comment:
Question for clarification

Reply:
There are a handful breaches that are keyed on phone numbers, but it's mostly email addresses. Breaches without either would indeed probably not be loaded into HIBP since there's no way to search for your records.

Reply:
Then I think we should either drop this sentence, or change it to:

> The email addresses of users are leaked in ever breach that HaveIPeenPwned tracks since that is what it uses to allow users to query it's service.

Or something like that. That is drop emphasis on _every_ and explain more.